src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] The frontend project has no tests.  Please, kindly delete the default test included with react, and write your own testing your components and Redux.

_In the industry, tests are written after any feature is added._ 

That way you will ensure your app's integrity. 💪

Reply:
Thanks for your your review @Meltrust. We are trying our best to do it.